### DISCUSSION TITLE
Download is not working in github actions

### DISCUSSION THREAD

**Discussion by ramirobg94** [getgauge/taiko#1796] [Development]
I am trying to build an e2e test that downloads a file by clicking on a link.

When I run locally and in dockers it works but when running the tests into GitHub actions I am not able to see the file after the download.

I used this, 

await client().send('Page.setDownloadBehavior', {
      behavior: 'allow',
      downloadPath,
    })
    
    but the downloadPath doesn't contain the fail, 
    
    any idea? experience?
    
    Thanks,

**Comment by sriv** [comment]
Is your action public? I can take a look if it is. If not, could you share a sample illustrating the issue?

**Comment by KarianneM** [comment]
downloadPath: downloadPath is what I used
